### PR TITLE
Downgrade Hugo Version

### LINKED ISSUE #7525: Hugo default template builds are failing
### Step to reproduce:-

- Go to https://vercel.com/new/clone?s=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Ftree%2Fmain%2Fexamples%2Fhugo&template=hugo&id=67753070&b=main&from=templates
- Setup the project

### Result:

image

`WARN 2022/03/06 07:07:25 Module "ananke" is not compatible with this Hugo version; run "hugo mod graph" for more information.
Error: add site dependencies: load resources: loading templates: "/vercel/path0/themes/ananke/layouts/partials/func/warn.html:15:1": parse failed: template: partials/func/warn.html:15: function "warnf" not defined
Error: Command "hugo -D --gc" exited with 255`

### PR DESCRIPTION
**Pull request by leerob**
This was the minimum version supported by the `anake` theme we updated. It's still newer than the previous (and you can always override), but it's not such a drastic jump.

Closes https://github.com/vercel/vercel/issues/7525

### CODE REVIEW

**Review by AndyBitz — APPROVED** [review]

**Comment by leerob** [comment]
@AndyBitz mind reviewing again? Might have been a flakey test.

**Review by AndyBitz — APPROVED** [review]
@leerob Yes, that was a flakey test, feel free to merge it

File: packages/frameworks/src/frameworks.ts [hunk 1]
```diff
@@ -1772,7 +1772,7 @@ export const frameworks = [
 
       return (config && config.publishDir) || 'public';
     },
-    defaultVersion: '0.92.2',
+    defaultVersion: '0.64.0',
```
Comment (styfle) [inline packages/frameworks/src/frameworks.ts:1775]: This version isn't correct. The default version shipped in the build image is `0.58.2`

Comment (leerob) [reply packages/frameworks/src/frameworks.ts:1775]: Hmm - maybe this is the root issue - where is that at?

Comment (styfle) [reply packages/frameworks/src/frameworks.ts:1775]: Its in the `build-container`

**Pull request merged**